### ISSUE TITLE
Newsletter unreadable native iOS email client

### ISSUE THREAD

**Issue by dhruvb14** [tidusjar/Ombi#1009]
#### Ombi Version:

V 2.0.1234

#### Update Branch:

Stable

#### Operating System:

Docker

#### Mono Version (only if your not on windows)

4.6.2 (Stable 4.6.2.16/ac9e222 Tue Jan 3 11:48:26 UTC 2017)

#### Problem Description:

Whenever you view the newsletter email on iOS native email client you can not see any of the images, text or contents of the email below the plexrequests logo. Also issue when viewing in the "Inbox by Google" app where half way down the email the tables shift and the email becomes partially unreadable because the content overflows the view space. 

#### Reproduction Steps:

Send test admin email and view in iOS email client, Inbox by Google client and on desktop to compare differences
![image](https://cloud.githubusercontent.com/assets/4459649/22388824/8b6bd3c6-e4b0-11e6-8a84-fa8c3cf0e7f4.png)

**Comment by tidusjar** [comment]
Well that sucks! I hate dealing with emails and email clients lol.

Thanks for reporting this. I'll take a look when I redesign the newsletter

**Comment by dhruvb14** [comment]
Hey @tidusjar its probably just some bad html. I found the RecentlyAdded service thats make the newsletter however whenever I build the solution, I'm getting 1x branch. Is main not where 2x code lives? I was hoping to help you out and see if i could take a stab at it.

**Comment by tidusjar** [comment]
Yeah so the HTML is generated in the `Ombi.Services.Jobs.RecentlyAdded` class.

I'm not sure what you mean by your are getting something different? If you switch to the dev branch it should be fine.

**Comment by dhruvb14** [comment]
Ok ill try switching to dev branch, was just confused why when I pulled main branch and ran it that 1.0 still ran and not 2.0 since my docker image pulls from main and has 2.0 running.

**Comment by tidusjar** [comment]
Ohhh you mean the version number! Ignore that when developing locally, the way the version number gets generated is that it's part of the assembly version, the assembly version is only correct when going through the official build system, so if you run it locally it will always show as version 1.0

**Comment by dhruvb14** [comment]
Not sure if you were working on this, but I have figured out what the issue is, its an issue with using <hr> tags outside of a <td> tag. I have made progress to get rid of the 30-40 <hr> tags that were being added to the top of the email and gotten it down to about 2 which I can't find where they are being injected. I am likely going to create a few classes to make the code a bit cleaner and easier to see visually as the html is being created. Just wanted to give you a heads up that you don't need to work on this, I hope to have a PR to you tomorrow evening.

![image](https://cloud.githubusercontent.com/assets/4459649/22394419/ba362236-e4ee-11e6-9551-f1454fcd4f6c.png)
